### PR TITLE
chore(deps-dev): bump next from 10.0.0 to 11.0.0

### PR DESCRIPTION
**Pull request by notjustinshaw**
Bumps NextJS from v10 to v11
----
Solves https://github.com/yonycalsin/nextjs-cors/issues/59

NextJS v11.0 Upgrade Guide: https://github.com/vercel/next.js/blob/canary/docs/upgrading.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +60,6 @@
       "typescript": "^4.2.2"
    },
    "peerDependencies": {
-      "next": "^8.1.1-canary.54 || ^9.0.0 || ^10.0.0"
+      "next": "^8.1.1-canary.54 || ^9.0.0 || ^10.0.0 || ^11.0.0"
```
Comment (notjustinshaw) [inline package.json]: @yonycalsin - this should be all we need to resolve #59, unless there are conflicts elsewhere.

**Comment by yonycalsin** [comment]
thanks for the PR I just updated it in a new PR along with other dependencies. https://github.com/yonycalsin/nextjs-cors/pull/61

**Comment by yonycalsin** [comment]
thanks 🎉  @justinalexandershaw

**Comment by notjustinshaw** [comment]
No, thank you!

**Comment by sorenrood** [comment]
@justinalexandershaw 👑

**Pull request merged**